### ISSUE TITLE
v1.0.0 package doesn't have /lib folder

### ISSUE THREAD

**Issue by cailborg** [dreipol/storybook-figma-addon#10]
There's no `/lib` folder in your package anymore, so the import in `register.js` is failing

```
ModuleNotFoundError: Module not found: Error: Can't resolve './lib/register' in '/***/node_modules/@dreipol/storybook-figma-addon'

**Comment by GianlucaGuarini** [comment]
Please open a PR to fix the issue

**Comment by cailborg** [comment]
@GianlucaGuarini I can build it, but your gitignore is set to ignore `/lib`.
I think you need to build it before you release a new package version.

**Comment by GianlucaGuarini** [comment]
Fixed in 1.0.1 thank you for your feedback
